src/main/drivers/accgyro/accgyro_mpu6500.c

```diff
@@ -108,3 +113,5 @@ bool mpu6500GyroDetect(gyroDev_t *gyro)
 
     return true;
 }
+
+#endif
```

Comment:
```suggestion
#endif   // USE_...
```

Reply:
Yes this one is dirty. Needed SPI defines too